src/WebStarter/appsettings.json

```diff
@@ -217,11 +229,11 @@
   },
 
   "Database": {
-    "Default": "FileRepository",
+    "Default": "MongoRepository",
     "TablePrefix": "BotSharp",
-    "BotSharpMongoDb": "",
+    "BotSharpMongoDb": "mongodb://localhost:27017/via-aihub",
     "FileRepository": "data",
-    "Assemblies": [ "BotSharp.Core" ]
+    "Assemblies": [ "BotSharp.Plugin.MongoStorage" ]
```

Comment:
This is not correct. It should reference the project that contains the table and collection definitions.